tests/jballerina-unit-test/src/test/resources/test-src/query/simple-query-with-var-type.bal

```diff
@@ -390,7 +390,7 @@ function testQueryConstructingTableWithVar() returns error? {
                   where user.age > 21 && user.age < 60
                   select {user};
 
-    assertEquality(true, result1 is table<record {| User user; |}> key(user));
+    assertEquality(true, result1 is table<record {| User user; |}>);
```

Comment:
Why are these changes needed?

Reply:
To define a key that field should be read-only. right? Due to this change(https://github.com/ballerina-platform/ballerina-lang/blob/21d5a766e8b4592933177c1c252bf84413a3ce89/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java#L5580) I fixed this test case.